src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -280,6 +303,18 @@ export default class GroupComparisonPage extends React.Component<IGroupCompariso
                         >
                             Save to new comparison session
                         </button>
+                        <button
+                            className="btn btn-xs btn-default"
+                            onClick={this.store.clearDragUidOrder}
+                        >
+                            Reset
+                        </button>
+                    </div>
+                    <div className="btn btn-xs btn-none"
```

Comment:
why btn if we're not using any btn styles?  i mean, why not just ```<i>```

Reply:
(1) we need a div not just <i> because otherwise the click hitzone is just the actual content of the icon, not the entire block around it
(2) there are still styles applied on mouse down